src/Controls/src/Core/Platform/Android/Extensions/ToolbarExtensions.cs

```diff
@@ -147,6 +147,9 @@ public static void DisposeMenuItems(this AToolbar toolbar, IEnumerable<ToolbarIt
 
 		internal static void UpdateMenuItemIcon(IMauiContext mauiContext, IMenuItem menuItem, ToolbarItem toolBarItem, Color tintColor)
 		{
+			if (toolBarItem.IconImageSource == null)
```

Comment:
I think the extension method should be modified to accept a null ImageSource and then it just creates a result with a null value that gets passed into the callback

Reply:
Can you make a PR with that? 
I will merge this now just to fix running android